README.md

```diff
@@ -1,5 +1,6 @@
 # Dark Theme for Jenkins (Incubated project)
 
+[![Changelog](https://img.shields.io/github/release/jenkinsci/dark-theme.svg?label=changelog)](https://github.com/jenkinsci/dark-theme/releases/)
```

Comment:
do we need it here and further down? 

it's already there to show the latest version by the cdn link

Reply:
The trick is in accessibility of the badge from the top which is a shortcut for the changelog thanks to the label. The link below just refers a recommended version, not a changelog